x-pack/plugins/apm/public/components/app/ErrorGroupDetails/index.tsx

```diff
@@ -88,24 +89,10 @@ export function ErrorGroupDetails({ location, match }: ErrorGroupDetailsProps) {
     }
   }, [serviceName, start, end, groupId, uiFilters]);
 
-  const { data: errorDistributionData } = useFetcher(() => {
-    if (start && end) {
-      return callApmApi({
-        endpoint: 'GET /api/apm/services/{serviceName}/errors/distribution',
-        params: {
-          path: {
-            serviceName,
-          },
-          query: {
-            start,
-            end,
-            groupId,
-            uiFilters: JSON.stringify(uiFilters),
-          },
-        },
-      });
-    }
-  }, [serviceName, start, end, groupId, uiFilters]);
+  const { errorDistributionData } = useErrorGroupDistributionFetcher({
+    serviceName,
+    groupId,
+  });
```

Comment:
This was extracted to a separate hook to allow two components to re-use the same logic.